### ISSUE TITLE
wrap demo in docker compose

### ISSUE THREAD

**Issue by shuttie** [metarank/metarank#462]
so you can replicate the demo with only `docker compose up`

**Comment by ChakshuGautam** [comment]
Of course. I can contribute to a [tilt file](https://github.com/tilt-dev/tilt) for the chart as well, if that is something that would make the dev setup easier.
